src/Commands/AbstractDrupalCommands.php

```diff
@@ -316,8 +317,46 @@ public function drushSetup(array $options = [
     }
 
     /**
-     * Setup Drupal settings overrides.
+     * Setup Drupal services file.
+     *
+     * This command will add the configuration under service_parameters present in
+     * runner.yml into "services.yml" file in the site directory.
+     *
+     * @command drupal:services-setup
+     *
+     * @option root         Drupal root.
+     * @option sites-subdir Drupal site subdirectory.
+     * @option force        Drupal force generation of a new services.yml.
+     *
+     * @param array $options
      *
+     * @return \Robo\Collection\CollectionBuilder
+     */
+    public function servicesSetup(array $options = [
+        'root' => InputOption::VALUE_REQUIRED,
+        'sites-subdir' => InputOption::VALUE_REQUIRED,
+        'force' => false,
+    ])
+    {
+        // Read given parameters.
+        $service_parameters['parameters'] = $this->getConfig()->get('drupal.service_parameters', []);
+        $dumper = new Dumper(2);
+        $yaml = $dumper->dump($service_parameters, PHP_INT_MAX, 0, Yaml::DUMP_EXCEPTION_ON_INVALID_TYPE);
```

Comment:
Since this basically prints out a YAML file as Drupal expects to let's move this two lines in a separated protected method, so it can be used on all Drupal commands.